go-indexer/indexer/transformer.go

```
@@ -132,85 +145,78 @@ func (t *Transformer) Start() error {
 
-					t.logger.Debug("Committing transaction for event: %v", event.Id)
-					if err := tx.Commit(); err != nil {
-						t.logger.Error("Error committing transaction: %v", err)
-						t.handleError(err)
-						break
-					}
-					t.logger.Info("Successfully committed transformation for event: %v", event.Id)
+		dbOperations, err := t.transform(event)
+		if err != nil {
+			t.logger.Error("Error computing transformation: %v", err)
+			tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)